server/src/test/scala/controllers/UsersControllerSpec.scala

```diff
@@ -284,4 +285,203 @@ class UsersControllerSpec extends PlayPostgresSpec with LoginUtils {
     }
 
   }
+
+  "POST /forgot-password" should {
+    "create the reset password token after the user's request to reset their password" in withApiClient { client =>
+      val name = Name.trusted("wiringbits")
+      val email = Email.trusted("test1@email.com")
+      val request = CreateUser.Request(
+        name = name,
+        email = email,
+        password = Password.trusted("test123..."),
+        captcha = Captcha.trusted("test")
+      )
+
+      createVerifyLoginUser(request, client).futureValue
+
+      val forgotPasswordRequest = ForgotPassword.Request(email, Captcha.trusted("test"))
+      val response = client.forgotPassword(forgotPasswordRequest).futureValue
+
+      response must be(ForgotPassword.Response())
+    }
+
+    "ignore the request when the user tries to reset a password for unknown email" in withApiClient { client =>
```

Comment:
This is unnecessary, we need to ignore the request when the user clicks forgets its password but such an email doesn't exists.

Reply:
Actually, I got confused, this is ok.